src/model.ts

```diff
@@ -331,7 +334,7 @@ export function convertDescribeJobtoJobDetail(
   return {
     ...emptyCreateJobModel(),
     jobId: describeJob.job_id,
-    jobName: describeJob.name ?? '',
+    jobName: MakeNameValid(describeJob.name ?? ''),
```

Comment:
The name returned from the backend will always be valid, so please revert the changes in this file to avoid conflicts with #258.